### PR TITLE
[exporter/loki] Document the migration from the Loki Exporter to the Loki V3 OTLP endpoint

### PR DESCRIPTION
**Pull request by cyrille-leclerc**
**Description:** <Describe what has changed.>

Document how to migrate from the now deprecated OTel Collector Loki Exporter to the new Loki V3 OTLP endpoint and new OTel log format

### CODE REVIEW

File: exporter/lokiexporter/README.md [hunk 1]
```diff
@@ -17,7 +17,201 @@ Exports data via HTTP to [Loki](https://grafana.com/docs/loki/latest/).
 
 This component is **deprecated**: Loki now supports native [OTLP ingestion](https://grafana.com/docs/loki/latest/send-data/otel/) starting from v3. Grafana Cloud also supports [OTLP native ingestion](https://grafana.com/docs/grafana-cloud/send-data/otlp/send-data-otlp) for logs. This component will be removed in September 2024.
```
Comment (crobert-1) [inline exporter/lokiexporter/README.md:18]: From this section of the README the plan is to remove this entire exporter in the next month. Why do we want to update the README for a week or two if we're just going to delete this entire component one release later (at most)? Just want to make sure I understand what's happening here, and the best way to communicate this to users. 👍

Comment (cyrille-leclerc) [reply exporter/lokiexporter/README.md]: Great catch @crobert-1 ,
As discussed with @jpkrohling , we are postponing of 2 months the removal of the Loki Exporter from the OTel Collector Contrib to give users more time to migrate to the new native OTLP endpoint.
I updated this PR to change "September" to "November".

File: exporter/lokiexporter/README.md [hunk 2]
```diff
@@ -15,9 +15,203 @@ Exports data via HTTP to [Loki](https://grafana.com/docs/loki/latest/).
...
+
+#### Tempo data source: Trace 2 Logs
+
+To enable the "trace to logs" navigation from Tempo to Loki, navigate to the Grafana Tempo data source configuration screen, in the "Trace to logs" section, select "use custom query" and specify the query:
+
+```
+{service_name="${__span.tags["service.name"]}"} | trace_id="${__span.traceId}"
+```
+
+#### Loki data source: Log 2 Trace
```
Comment (crobert-1) [inline exporter/lokiexporter/README.md:197]:
```suggestion
#### Loki data source: Log to Trace
```
I can't find any reference to `2` being included in documentation, but let me know if I missed something.

**Review by crobert-1 — COMMENTED** [review]
I'm not very familiar with Loki, so I can't validate any functionality mentioned, my comments are simply formatting nits. Feel free to drop any you don't agree with.

Also, I noticed there's some overlap between what's included here and official Loki documentation, like [the link you've included here](https://grafana.com/docs/loki/latest/send-data/otel/). Is there any way we could just link to that page for overlapping information, rather than copying its contents here?

**Comment by cyrille-leclerc** [comment]
Thank you very much for your thorough review Curtis, it's extremely useful to me.


> Also, I noticed there's some overlap between what's included here and official Loki documentation, like [the link you've included here](https://grafana.com/docs/loki/latest/send-data/otel/). Is there any way we could just link to that page for overlapping information, rather than copying its contents here?

Our rationale o publish an end to end migration guide for OTel Collector users is twofold:
* Our tech writers recommend to write are self contained docs rather than requiring to switch back and forth between multiple doc pages, especially when spanning across multiple websites.
* Our documentation page [Loki > Send Data > Ingesting logs to Loki using OpenTelemetry Collector](https://grafana.com/docs/loki/latest/send-data/otel/) is not intended to be a migration guide but focuses on new OTel Collector setups.

Please check me @jpkrohling

**Review by jpkrohling — APPROVED** [review]
LGTM, would be good to have a review from someone working on Loki.

**Review by sandeepsukhani — APPROVED** [review]
Left some minor nits otherwise it LGTM

**Pull request merged**